docs/kns_apps_tutorial.md

```diff
@@ -0,0 +1,347 @@
+# Jarvice KNS Applications tutorial
+
+![GlobalProcess](img/apps_tutorial/SuperNimbix.svg)
+
+Note: it is assumed that the user is already familliar with Jarvice Push to Compute (P2C). Please refer to [apps tutorial](apps_tutorial.md).
```

Comment:
Note: It is assumed that the user is familiar with Jarvice Push to Compute (P2C). For more information, please refer to [apps tutorial](apps_tutorial.md).